tests/Unit/Time/StepChoosers/Test_Cfl.cpp

```diff
@@ -55,7 +55,7 @@ struct Metavariables {
 double get_suggestion(const size_t stepper_order, const double safety_factor,
                       const double characteristic_speed,
                       const DataVector& coordinates) noexcept {
-  const Parallel::GlobalCache<Metavariables> cache{{}};
+  const Parallel::GlobalCache<Metavariables> cache{};
```

Comment:
Just for my understanding (and then maybe a suggestion): this calls a contructor with a `nullptr` secretly? And then what sort of an error does one get? If it's not clear could be add an 'ERROR' in a constructor to make things easier to debug? I don't know what the current error was, so I'm just checking that it is reasonably easy to debug :)

Reply:
Right, so my understanding from the error was that it was calling the `CkMigrateMessage*` constructor with a `nullptr`, and doing so was fine when we were forgetting to pass to `CBase...`, but the `CBase...` constructors error with some kind of terrible segfault when you hand them a malformed migrate message, so ... not super easy to debug.
I'll add an `ERROR` for the `nullptr` case and a documentation note. It doesn't actually catch all the problems, because even a valid pointer to an incorrectly constructed message object also led me to a segfault :(.